torchci/lib/drciUtils.ts

```diff
@@ -24,14 +25,13 @@ Note: Links to docs will display an error until the docs builds have been comple
 
 export function formDrciComment(
     pr_num: number, 
-    pr_results?: string,
+    pr_results: string,
```

Comment:
we just pass in "" everywhere right? Maybe default this to that or get rid of it?